docs/config_search.md

```diff
@@ -36,13 +37,14 @@ Model Analyzer's `profile` subcommand supports multiple modes when searching to
 - [Brute Force Search](config_search.md#brute-search-mode)
   - **Search type:** Brute-force sweep of the cross product of all possible configurations
   - **Default for:**
-    - Single non-ensemble models
+    - Single non-ensemble/BLS models
```

Comment:
I read this as single non-ensemble models and single BLS models

Reply:
Fixed